templates/components/products/list-item.html

```diff
@@ -4,6 +4,13 @@
     <article class="listItem">
 {{/if}}
     <figure class="listItem-figure">
+        <a href="{{url}}"
+           class="listItem-figure__link"
+           aria-label="{{name}},{{> components/products/product-aria-label}}"
+           {{#if settings.data_tag_enabled}}
```

Comment:
What is this setting responsible for?

Reply:
For image to be a link to a product (see first screen)

Reply:
What is the diff between existing of setting or not?

Reply:
Without this link user will not have possibility to click on image to be redirected on PDP